VogtObserver2/Program.cs

```diff
@@ -1,12 +1,58 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Threading;
 
 namespace VogtObserver2
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            
+
+            Run();
+        }
+
+        public static void Run()
+        {
+            StockGrabber stockGrabber = new StockGrabber();
+
+            Trader t1 = new Trader("Julie");
+            Trader t2 = new Trader("Amy");
+            Trader t3 = new Trader("Mark");
+
+            stockGrabber.AddObserver(t1);
+            stockGrabber.AddObserver(t2);
+            stockGrabber.AddObserver(t3);
+
+
+            for (int i = 0; i <= 5; i++)
+            {
+
+                stockGrabber.FetchStockPrice(GetRandomStockSymbol());
+
+                Thread.Sleep(2000);
+
+            }
+        }
+
+        public static string GetRandomStockSymbol()
+        {
+
+            string[] tempSymbols = new string[3] { "GOOG", "MSFT", "AAPL" };
+            string symbol = " ";
+
+            Random random = new Random();
```

Comment:
You should generally assign random to the class and reuse the same variable. Not tooo big of a deal, but something you'd want to do in the real world. In fact, I think the recommendation is to create 1 Random class and use it throughout the entire program. Can't recall -- but might be worth googling and reading about best practices and why you would do that. A minor detail though I wouldnt expect an entry developer to know immediately.